addons/mail/static/src/core/common/chat_window.js

```diff
@@ -132,6 +132,12 @@ export class ChatWindow extends Component {
                 }
                 break;
             }
+            case "k":
+                if (ev.ctrlKey) {
+                    this.store.env.services.command.openMainPalette({ searchValue: "@" });
+                    ev.preventDefault();
+                }
+                break;
```

Comment:
This does't work with macos: ctrl+k is actually cmd+k.

Fix here looks fine, but wondering if we couldn't do better, like hotkey service could expose the global registrations hotkeys, so that we can not `ev.stopPropagation();` when this is not a global hotkey (so that we just bully the home menu, not the hotkey service)

Reply:
See `getActiveHotkey` usage like this ongoing PR that does the same in discuss app: https://github.com/odoo/odoo/pull/199195/files